store/src/java/com/zimbra/cs/mailbox/ContactAutoComplete.java

```diff
@@ -409,6 +409,7 @@ public AutoCompleteResult resolveEmailAddr(String str) throws ServiceException {
              }
            return result;
     }
+
```

Comment:
Revert the change so that the file does not appear in code diff (or Files changed tab in the PR).

Reply:
Done.